python/private/py_executable.bzl

```diff
@@ -1958,8 +1960,8 @@ def create_executable_rule_builder(implementation, **kwargs):
         ],
         cfg = dict(
             implementation = _transition_executable_impl,
-            inputs = [_PYTHON_VERSION_FLAG],
-            outputs = [_PYTHON_VERSION_FLAG],
+            inputs = TRANSITION_LABELS + [_PYTHON_VERSION_FLAG],
+            outputs = TRANSITION_LABELS + [_PYTHON_VERSION_FLAG],
```

Comment:
nit: the `_PYTHON_VERSION_FLAG` redefines what is defined in `common_labels.bzl`. This could be also addressed as a followup.

Reply:
Thanks, yeah, something to address in a follow up.